services/core/CrateHistorian/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _Crate-Historian:
+
+===============
+Crate Historian
+===============
+
+Crate is an open source SQL database designed on top of a No-SQL design.  It
+allows automatic data replication and self-healing clusters for high
+availability, automatic sharding, and fast joins, aggregations and sub-selects.
+
+Find out more about crate from `<https://crate.io/>`_.
+
+Note
+~~~~
+CrateHistorian is still alpha, schemas could change in the future, do not use
+this for production data until schema is confirmed as final
+Currently the historian supports two schemas for numerical data, the primary
+schema closely resembles the SQLHistorian schema but there is an optional
+"raw" schema that can be enabled in the config below that utilizes some of
+the advanced indexing features of crate
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+1. Crate Database
+-----------------
+
+For Arch Linux, Debian, RedHat Enterprise Linux and Ubuntu distributions there
+is a simple installer to get crate up and running on your system.
+
+    ::
+
+        sudo bash -c "$(curl -L install.crate.io)"
+
+This command will download and install all of the requirements for running
+crate, create a crate user and install a crate service.  After the installation
+the service will be available for viewing at http://localhost:4200 by default.
+
+.. note::  There is no authentication support within crate.
```

Comment:
Authentication for crate is an enterprise subscription only feature.

Reply:
Done